dts/bindings/timer/st,stm32-timers.yaml

```diff
@@ -46,3 +46,32 @@ properties:
                                                down.
 
       If absent, then STM32_TIM_COUNTERMODE_UP is used (reset state).
+
+  st,master-mode:
```

Comment:
I'm a bit annoyed by the discrepancy vs `st,countermode` writing, even if the latter feels less nice.